spec/models/lead_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Lead, type: :model do
+
+it "validates uniqueness of lead" do
+  FactoryGirl.create(:lead)
+    should validate_uniqueness_of(:company_id).case_insensitive
+  end
```

Comment:
`end` at 8, 2 is not aligned with `it "validates uniqueness of lead" do` at 5, 0.